test/init-mocha-webdriver.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Taken from Grist, only including the variables that seems the most relevant to weasel. In particular it allow to skip the ` env MOCHA_WEBDRIVER_IGNORE_CHROME_VERSION=1` bit when running test.